ios/Exponent/Versioned/Core/EXVersionManager.mm

```diff
@@ -446,4 +455,24 @@ - (BOOL)_isOpeningHomeInProductionMode
   return _params[@"browserModuleClass"] && !_params[@"manifest"][@"developer"];
 }
 
+- (void *)versionedJsExecutorFactoryForBridge:(RCTBridge *)bridge
+{
+  __weak __typeof(self) weakSelf = self;
```

Comment:
I'm not sure how Objective-C ↔️ Objective-Cpp Interoperability works, but is it possible to either use existing `WEKIFY` and `ENSURE_STRONGIFY` macros or create new one for Cpp realm?